src/test/java/uk/gov/hmcts/reform/hmc/validator/BeanValidatorTest.java

```diff
@@ -507,4 +508,37 @@ void whenInvalidPartyRoleIsEmpty() {
         assertTrue(validationErrors.contains(PARTY_ROLE_EMPTY));
     }
 
+    @Test
+    void whenValidPartyRoleIs40Characters() {
+        PartyDetails partyDetails = new PartyDetails();
+        partyDetails.setPartyID("XXX1");
+        partyDetails.setPartyType(PartyType.IND.getLabel());
+        partyDetails.setPartyRole("abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMN");
+        Set<ConstraintViolation<PartyDetails>> violations = validator.validate(partyDetails);
+        assertTrue(violations.isEmpty());
+        List<String> validationErrors = new ArrayList<>();
+        violations.forEach(e -> {
+            validationErrors.add(e.getMessage());
+            logger.info(e.getMessage());
+        });
+        assertEquals(0, violations.size());
+    }
+
+    @Test
+    void whenInvalidPartyRoleIs41Characters() {
+        PartyDetails partyDetails = new PartyDetails();
+        partyDetails.setPartyID("XXX1");
+        partyDetails.setPartyType(PartyType.IND.getLabel());
+        partyDetails.setPartyRole("abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNO");
+        Set<ConstraintViolation<PartyDetails>> violations = validator.validate(partyDetails);
+        assertFalse(violations.isEmpty());
```

Comment:
Could reduce lines 534 - 541 down to

```
assertEquals(1, violations.size());

assertTrue(violations.stream()
    .map(ConstraintViolation::getMessage)
    .anyMatch(msg -> msg.equals(PARTY_ROLE_MAX_LENGTH)));
```

Reply:
done

Reply:
Updated in 96433765a84f6a5fe58bb8429db9c033c06eeaeb